web/client/codechecker_client/cmd_line_client.py

```diff
@@ -1324,7 +1324,14 @@ def print_reports(client,
     client = None
```

Comment:
It would be good to print out the matching local report directories for basename and newname options so the user would see the process what we are trying to diff:
```suggestion
    if basename_local_dirs:
        LOG.info("Matching local report directories (--baseline): %s",
                 basename_local_dirs)

    if newname_local_dirs:
        LOG.info("Matching local report directories (--newname): %s",
                 newname_local_dirs)

    client = None
```

Reply:
Done.